### PR TITLE
Week 1 elias tsegay

### PR DESCRIPTION
**Pull request by etsegay**

### CODE REVIEW

File: coding-concepts.js [hunk 1]
```diff
@@ -8,44 +8,51 @@
...
 var weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+//console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:Error
+// b) Verify and explain:It is verified by runing it in teh terminal.The result was as expected
+//                      that is Error because toUpperCase() is a string method while the 
+//                      given data type is object.
```
Comment (WalkerTexas-coder) [inline coding-concepts.js:47]: well done!

File: interview-questions.md [hunk 2]
```diff
@@ -7,60 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer:numeric comparisons always return boolean values.when we compare numbers using logical operators like
+  >,<.===,==,!= etc
 
-  Researched answer:
+  Researched answer:Logical operators when used in logical comparisons usually return boolean values but some time when the && and || logical operators are used with non logical values ,they may return non boolean values.example console.log(0||null) will return none boolean value the null.
```
Comment (WalkerTexas-coder) [inline interview-questions.md:21]: This answer will do much better in an interview! good work!

File: interview-questions.md [hunk 3]
```diff
@@ -7,60 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
 
 
 4. What is a built-in method?
 
-  Your answer:
+  Your answer:Built-in method is a function developed by javascript developers and embeded to the javascript engine
 
-  Researched answer:
+  Researched answer:There are many built in methods in javascript,when the stored data in an object is a function,it is called a method.Method is what an object does while property is what an object has.
```
Comment (WalkerTexas-coder) [inline interview-questions.md:37]: Yes, a method is a function that belongs to and manipulates the object it is inside of.

File: code-challenges.js [hunk 4]
```diff
@@ -6,43 +6,100 @@
...
+Algorithm:subtract
+        input=> given two numbers
+        output=> difference of the two numbers
+
+        take a larger number and subtract the smaller number
+
+*/
+let number1=prompt("Enter a number")
+let number2=prompt("Enter another number")
+console.log(number1-number2)
```
Comment (WalkerTexas-coder) [inline code-challenges.js:82]: This is a good start but remember the prompt is asking for use to set up a situation where we must determine the greater number and place it as the first of two in the equation.

File: code-challenges.js [hunk 5]
```diff
@@ -6,43 +6,100 @@
...
+let boilingPoint=212
+if(temp<boilingPoint){
+    console.log(`${temp} is below boiling point`)
+}
+else if(temp == boilingPoint){
+    console.log(`${temp} is at boiling point`)
+}
+else{
+    console.log(`${temp} is above boiling point`)
+}
```
Comment (WalkerTexas-coder) [inline code-challenges.js:31]: Well done!

**Review by WalkerTexas-coder — APPROVED** [review]
Elias, Good work! I can see you have some javascript skills already. That can be very helpful in a class like this where we are starting from the basics. But it also creates a temptation to not read the prompts carefully since we feel like we already the the answer. Keep a close eye on the prompts and directions. It will matter a great deal when you are at work! Your PR is approved!

**Comment by etsegay** [comment]
Thank you so much for your feedback.
Yes I have little experience about JavaScript, I have gained a lot from
learn already specially GitHub and pair programming.

**Pull request merged**